topics/dev/tutorials/tool-annotation/tutorial.md

```diff
@@ -0,0 +1,208 @@
+> 2. Select **Add a tool** from the drop-down **Menu**
+> 3. Fill in general information in the first tab of the wizard. Required information includes:
+>    1. **Tool name** 
+>    2. **Description**
+>    3. **Homepage URL**
+> 4. Add EDAM operation concepts, as well as EDAM data concepts for both inputs and outputs
+>    
+>    {% include topics/dev/tutorials/tool-annotation/add_edam_function.md %}
+> 
+> 5. Add EDAM topic concepts
+>    
+>    {% include topics/dev/tutorials/tool-annotation/add_edam_topic.md %}
+> 
+> 6. Add the tool type, license, language, and other metadata as needed
+> 7. In the Permissions tab, select **Make this resource editable by anyone**
+> 8. Click on **Validate** on the top
+> 9. Click on **Save** to create the bio.tools entry
+> 10. Copy the bio.tools id
+{: .hands_on}
+
+</div>
+
+<div class="Existing-entry" markdown="1">
+
+# Review and update the EDAM terms for an existing bio.tools entry
+
+Before linking a Galaxy tool with its corresponding bio.tools entry, we need to check if the tool is correctly annotated with EDAM concepts.
+
+> <hands-on-title>Check EDAM terms in a bio.tools entry</hands-on-title>
+>
+> 1. Open the bio.tools entry for the tool
+> 2. Check the EDAM **Topic** terms by looking at the green boxes (if they exist) below the tool name, URL, and available versions
+> 3. Check the EDAM **Operation** terms by looking at the blue boxes (if they exist) below the tool description
+{: .hands_on}
+
+{% include _includes/cyoa-choices.html option1="Terms to be modified" option2="Correct terms" default="Terms to be modified" text="What do you think about the EDAM terms in the bio.tools entry?" disambiguation="edamUpdate"%}
+
+<div class="Terms-to-be-modified" markdown="1">
+
+To modify EDAM terms in a bio.tools entry, we need to request editing rights and then modify this entry.
+
+> <hands-on-title>Modify the EDAM concepts in a bio.tools entry</hands-on-title>
+>
+> 1. Sign up for [bio.tools](https://bio.tools/)
+> 2. Click on **Request editing rights** at the bottom of the bio.tools entry page
+> 3. Wait for the request to be approved
+> 4. Click on **Update this record**
+> 5. Update, or add, EDAM **Operation** term(s) and EDAM **Data** term(s) for both inputs and outputs
+>    
+>    {% include topics/dev/tutorials/tool-annotation/add_edam_function.md %}
+> 
+> 6. Update, or add, EDAM **Topic** term(s)
+>    
+>    {% include topics/dev/tutorials/tool-annotation/add_edam_topic.md %}
+> 7. Update, or add, the metadata for tool type, license, language, and other fields as needed
+> 8. Click on **Validate** on the top
+> 8. Click on **Save** to create the bio.tools entry
+> 9. Copy the bio.tools ID
+{: .hands_on}
+
+</div>
+
+</div>
+
+# Linking a Galaxy tool to a bio.tools entry
+
+To link a Galaxy tool to its corresponding bio.tools entry, we need to first find the source of the wrapper.
+
+> <hands-on-title>Find the Galaxy wrapper</hands-on-title>
+>
+> 1. Go to the tool on any Galaxy server
```

Comment:
I think we should use the parsed source folder instead: https://github.com/galaxyproject/galaxy_tool_metadata_extractor/pull/65
Which will be correct for all tools in our list.
For many tools I checked the location in the .shed file is wrong... that might confuse the helpers !